docs/develop/plugins/wasm/README.md

```diff
@@ -0,0 +1,210 @@
+---
+title: WASM Plugins
+children:
+  - assemblyscript.md
+  - rust.md
+  - go.md
+  - dotnet.md
+  - http_endpoints.md
+  - deltas.md
+  - capabilities.md
+  - best_practices.md
+---
+
+# WASM Plugin Development Guide
+
+## Overview
+
+This guide covers how to develop WASM/WASIX plugins for Signal K Server 3.0. WASM plugins run in a secure sandbox with isolated storage and capability-based permissions.
+
+## What Makes a WASM Plugin?
+
+A WASM plugin is identified by the **`wasmManifest`** field in `package.json`:
```

Comment:
```suggestion
A WASM plugin is an npm package that contains the WASM code for the plugin instead of the traditional JavaScript code. A WASM plugin is identified by the regular `signalk-node-server-plugin` keyword in package.json and the additional **`wasmManifest`** field in `package.json`:
```